frontend/.eslintrc.json

```diff
@@ -8,16 +8,16 @@
   "extends": [
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
-    "next/babel",
```

Comment:
**question (blocking):** Please, did you review that the tests are working as expected because `next/babel` is a dependency for Jest

Reply:
Good catch, `eslint` fails with `next/babel` so I replaced it with `next`. I'll test the changes with Jest now.

Reply:
Tests pass without `next/babel`